hypervisor/scenarios/industry/vm_configurations.c

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2019 Intel Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <vm_config.h>
+#include <vm_configurations.h>
+#include <acrn_common.h>
+#include <vuart.h>
+
+struct acrn_vm_config vm_configs[CONFIG_MAX_VM_NUM] = {
+	{
+		.type = SOS_VM,
+		.name = "ACRN SOS VM",
+		.uuid = {0xdbU, 0xbbU, 0xd4U, 0x34U, 0x7aU, 0x57U, 0x42U, 0x16U,	\
+			 0xa1U, 0x2cU, 0x22U, 0x01U, 0xf1U, 0xabU, 0x02U, 0x40U},
+			/* dbbbd434-7a57-4216-a12c-2201f1ab0240 */
+		.guest_flags = GUEST_FLAG_IO_COMPLETION_POLLING,
+		.clos = 0U,
+		.memory = {
+			.start_hpa = 0UL,
+			.size = CONFIG_SOS_RAM_SIZE,
+		},
+		.os_config = {
+			.name = "ACRN Service OS",
+		},
+		.vuart[0] = {
+			.type = VUART_LEGACY_PIO,
+			.addr.port_base = CONFIG_COM_BASE,
+			.irq = CONFIG_COM_IRQ,
+		},
+		.vuart[1] = {
+			.type = VUART_LEGACY_PIO,
+			.addr.port_base = INVALID_COM_BASE,
+		}
+	},
+	{
+		.type = NORMAL_VM,
```

Comment:
NORMAL_VM has been renamed to POST_LAUNCHED_VM.

Reply:
Thansk, rebased to the latest commit.

Reply:
@jsun26intel can u have a review?